packages/frontend/src/components/Home/index.js

```diff
@@ -1,21 +1,53 @@
 /* eslint-disable jsx-a11y/media-has-caption */
 import React, { useState } from 'react';
+import { useIdleTimer } from 'react-idle-timer';
 import VideoPlayer from '../VideoPlayer';
 import LegendItem from '../LegendItem';
 import Modal from '../Modal';
 
 function Home() {
+  // Array of video objects
+  const videos = [
+    { url: '/map-assets/one-day-loop.mp4', title: '24 hour loop' },
+    { url: '/map-assets/ten-day-loop.mp4', title: '10 day loop' },
+    { url: '/map-assets/one-year-loop.mp4', title: '1 year loop' },
+  ];
+
   // State to hold the current video URL
-  const [showMap, setShowMap] = useState(true);
-  const [showVideoPlayer, setShowVideoPlayer] = useState(false);
+  const [showMap, setShowMap] = useState(false);
+  const [showVideoPlayer, setShowVideoPlayer] = useState(true);
   const [showModal, setShowModal] = useState(false);
-  const [currentVideo, setCurrentVideo] = useState(false);
+  const [currentVideo, setCurrentVideo] = useState(videos[1].url);
+  const [startReset, setStartReset] = useState(false);
+
+  // Function to reload the page when idle
+  const onIdle = () => {
+    if (typeof window !== 'undefined') {
+      window.location.reload();
+    }
+  };
+
+  // Idle Timer Start
+  const { start } = useIdleTimer({
+    timeout: 1000 * 60 * 4, // 4 minutes
+    onIdle: () => onIdle(),
+    startManually: true,
+  });
```

Comment:
No need to wrap `onIdle` in another anonymous function:

```suggestion
  const { start } = useIdleTimer({
    timeout: 1000 * 60 * 4, // 4 minutes
    onIdle,
    startManually: true,
  });
```